### ISSUE TITLE
FTB Beyond Thermal Expansion Machines

### ISSUE THREAD

**Issue by Wolfsokim** [CoFH/Feedback#82]
First off, what I have been able to play with, awesome! Yesterday I logged into FTB Beyond (The initial release, not yet the update) and none of my machines were there. Or so I thought. They were still there but had become invisible blocks. They show up invisible in JEI as well. I'm not sure if this is a Thermal Expansion bug or some other thing, since only Thermal Expansion machines are showing this way. But I figured this was a good place to start. Screen shot to follow: 
![thermal expansion invisible machines](https://cloud.githubusercontent.com/assets/21130986/23826697/c1dba478-0656-11e7-9683-1708992cd291.jpg)

**Comment by KingLemming** [comment]
Try with the update please and get back to me.

Also if you could try with just CoFH mods, that'd be awesome.

And maybe post system spec.

This is an issue that we just don't experience first hand so...it's kind of a wtf.

**Comment by Wolfsokim** [comment]
I'm playing on a server, so I'll have to test that on my own in a different instance. I'll do that and get back to you.

**Comment by Wolfsokim** [comment]
The issue does not seem to occur in the update (I updated the entire FTB Beyond pack to the beta release). 

In the un-updated version I played around with it a bit and it seems to occur when I connect a ProjectRed Routed Interface Pipe to any of the machines, yet not all the time, either. 

I also was no able to recreate the issue with only CoFH mods installed.

![sysspecs](https://cloud.githubusercontent.com/assets/21130986/23827473/1fcc60e2-0669-11e7-9307-2cd6d3aeddd8.jpg)
GPU is AMD Radeon 7700 Series
